### ISSUE TITLE
在训练BERT时， Loss突然增大且模型无法继续学习

### ISSUE THREAD

**Issue by xlxwalex** [dbiir/UER-py#335]
我在训练BERT时使用了BookCorpus+Wikipedia-en数据，训练参数设置了batch_size=5120，warmup=0.1，learning_rate=4e-4，使用deep_init，没有用混合精度，steps(计算了40个epochs)=240k。但是在130k步左右突然Loss增大性能下降，且之后模型停止学习。请问这个可能是什么原因导致？ (Log如下所示)

![B0XI4S(0F6A~WNMT{EC8BJF](https://user-images.githubusercontent.com/26522478/183007327-7579d3a3-81ff-474a-8c56-22ae5a953a45.png)
之后模型就一直无法学习了
![33~Q)}R R54A)3U1V9BOC5I](https://user-images.githubusercontent.com/26522478/183007415-31c88792-fc19-4702-ac51-253e259ce835.png)

**Comment by hhou435** [comment]
模型用的哪个配置呢？可能是lr太大

**Comment by xlxwalex** [comment]
用的是Base的，因为batch_size比较大所以稍微放大了一点

**Comment by xlxwalex** [comment]
> 模型用的哪个配置呢？可能是lr太大

我尝试一下用更小的lr再试一下，谢谢您的回复！
